content/administration/install/deploy.rst

```diff
@@ -352,6 +352,32 @@ in ``/etc/nginx/sites-enabled/odoo.conf`` set:
     gzip on;
   }
 
+Enable HSTS
+-----------
+
+Once HTTPS enabled, an extra step to improve the security is to ensure you never
+transmit information over an unsecure connection. This can be done using `HTTP
+Strict Transport Security`_ or HSTS.
+
+On hosted odoo.com database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable HSTS on an Odoo instance, add the following element inside the `<head>`
+tag of the `web.layout` and `website.layout` view:
+
+.. code-block :: html
+
+  <link href="/_odoo/hsts/enable" rel="stylesheet"/>
```

Comment:
If we're talking about SaaS, this is done automatically for new databases since March this year. Besides I'm not sure we should recommend this approach, I think it was more of a workaround at the time.
Customers can requests to enable security features through our support now (can be done via the support page)
These security features bundle the following:
- Strict-Transport-Security (HSTS)
- Secure attribute on session_id cookie
- SameSite=Lax on session_id cookie